hoomd_lj/multistate_hoomd_lj.ipynb

```diff
@@ -21,7 +21,7 @@
    "outputs": [],
```

Comment:
<p>I think if you pin ipython to version 7.10.0, this message will be silenced.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mosdef-hub/mosdef-workflows/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
that is what i have found too. ill update the yml accordingly

Reply:
Should we disable warnings in the notebook? I would be concerned that pinning one particular dependency will break the environment file in the future (say when jupyterlab depends on a newer ipykernel)

Reply:
What would you prefer in this situation? Id be ok with either option. It probably makes sense to filter out the deprecation warning instead.

If so, please feel free to undo my commit and we can merge, all seems well otherwise

Reply:
I don't have a strong preference.

Reply:
Ok, lets instead filter the warning, unpin ipython, and when the ipykernel updates to the next major version, we can remove the filter _hopefully_